lms/migrations/versions/f68aacfc62c7_migration_for_grading_sync_tables.py

```diff
@@ -0,0 +1,51 @@
+"""Migration for grading_sync tables
+
+Revision ID: f68aacfc62c7
+Revises: adc83819c8d8
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = 'f68aacfc62c7'
+down_revision = 'adc83819c8d8'
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('grading_sync',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('assignment_id', sa.Integer(), nullable=False),
+    sa.Column('status', sa.Enum('scheduled', 'in_progress', 'finished', 'failed', name='autogradingsyncstatus', native_enum=False, length=64), nullable=False),
+    sa.Column('created', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+    sa.Column('updated', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+    sa.ForeignKeyConstraint(['assignment_id'], ['assignment.id'], name=op.f('fk__grading_sync__assignment_id__assignment')),
+    sa.PrimaryKeyConstraint('id', name=op.f('pk__grading_sync'))
+    )
+    op.create_index(op.f('ix__grading_sync_assignment_id'), 'grading_sync', ['assignment_id'], unique=False)
+    op.create_table('grading_sync_grade',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('grading_sync_id', sa.Integer(), nullable=False),
+    sa.Column('lms_user_id', sa.Integer(), nullable=False),
+    sa.Column('grade', sa.Float(), nullable=True),
```

Comment:
Grade is null when we "schedule" the grade sync. 

Having the rows in the DB simplifies the queries to know if we are already done, we can just query this table instead of having to store the expected result in the GradeSync table.

Reply:
Oh, I thought grade would hold the value that is supposed to be synced when the corresponding task runs. But if I'm getting what you say, this value is set later, once the actual sync with the LMS has succeeded?

If that is the case, how does the celery task know what is the grade that needs to be sent to the LMS?

Reply:
Yes, you are right, have done a few iterations of this locally and I was describing the previous one.

Grade should **not** be nullable here